CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-1.6.?? (2022-??-??)
+1.6.12 (2024-??-??)
+-------------------
+
+* Add support for PHP 8.2's `$content` property in `vfsStreamWrapper`
```

Comment:
Is this intentionally duplicated?

Reply:
Sorry, I forgot to update this. It's now corrected.